circuit-lib/circuit-lib.circom/scripts/buildCircuit.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env sh
```

Comment:
Can you `chmod +x` this file?

Only one script seems to have an executable bit.

```
╭───┬──────────────────────────────────────────────────────────────────┬──────┬────────┬──────────┬───────────┬
│ # │                               name                               │ type │ target │ readonly │   mode    │
├───┼──────────────────────────────────────────────────────────────────┼──────┼────────┼──────────┼───────────┼
│ 0 │ circuit-lib/circuit-lib.circom/scripts/buildCircuit.sh           │ file │        │ false    │ rw-r--r-- │
│ 1 │ circuit-lib/circuit-lib.circom/scripts/checkBuildCircuit.sh      │ file │        │ false    │ rwxr-xr-x │
│ 2 │ circuit-lib/circuit-lib.circom/scripts/createRustVerifyingKey.ts │ file │        │ false    │ rw-r--r-- │
╰───┴──────────────────────────────────────────────────────────────────┴──────┴────────┴──────────┴───────────┴

```